ui/appui-react/src/appui-react/UiFramework.ts

```diff
@@ -428,12 +428,16 @@ export class UiFramework {
 
   /** @internal */
   public static loggerCategory(obj: any): string {
-    const className = getClassName(obj);
+    const className = UiFramework.getObjectClassName(obj);
     const category =
       UiFramework.packageName + (className ? `.${className}` : "");
     return category;
   }
 
+  private static getObjectClassName(obj: any): string {
```

Comment:
Use this function from `core-react` (keep it internal), do not duplicate it everywhere.

Reply:
@raplemie @GerardasB Did that by importing the function directly - however having a tough time getting this to work with both mocha and storybook. Was the idea to keep internal but still export via barrel?

Reply:
Yes, export it via barrel so we dont mix esm and cjs since this is not only used in tests.

Reply:
Done, thanks.